structure/0006-governance.md

```diff
@@ -2,53 +2,161 @@
- * For a community moderator, this would be all community moderators and all members of the administrative board
+A team member is a member of the organization assigned to a specific team on GitHub.
+A member of the organization may be assigned to multiple teams.
+A team must have at least one member.
+
+### Team Leads
+
+A team lead is a team member with the [Team Maintainer](https://docs.github.com/en/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member#about-team-maintainers) 
+role for the relevant team on GitHub.
+A member of the organization may be a team lead for multiple teams.
+Each team must have at least one team lead.
+A team lead may be nominated via general consensus of the team, or by an election, the process being detailed below.
+
+### Team Responsibilities
+
+Team responsibilities will vary and be detailed on a team-by-team basis.
+In general, a team is responsible for performing the tasks outlined in the RFC that proposed its creation.
+
+Most teams will also maintain one or more repositories on GitHub.
+The team is responsible for reviewing pull requests and ensuring that they meet The Quilt Project's goals in quality and purpose.
+Once a team has decided that a pull request is suitable for merging, a team lead will be responsible for the actual merging. Team leads can also decide to allow some or all team members to perform merges.
+
+### Creating a Team
+
+A new team can be created by passing an RFC proposing that team.
+The initial team leads must be specified in the pull request adding the RFC and must approve the RFC before it is merged.
+Once the [RFC process](0001-rfc-process.md) is concluded and the RFC is merged,
+a new team with the specified leads will be created on GitHub by the Administrative Board.
+
+The RFC proposing the new team should contain the following major sections:
+
+- **Motivation**:
+  - Why is this team needed?
+  - What problems will this team solve?
+  - What tasks will this team be responsible for?
+
+- **Explanation**:
+  - What will this team *do*?
+  - What new projects will be created that this team will be responsible for?
+  - How will each of these projects benefit the Quilt organization/community?
+  - What processes will this team follow for each of the different kinds of changes their projects may undergo?
+
+- **Drawbacks**:
+  - Is there overlap of responsibilities?
+  - Is there excessive granularity?
+
+- **Rationale and Alternatives**:
+  - Is there already a team responsible for the described tasks?
+  - If this RFC doesn't pass, which team would be responsible for the described tasks?
+
+- **Prior Art**:
+  - Are there other organizations or projects with comparable tasks?
+
+### Sub Teams
+
+Some teams may wish for more granular organization of their structure.
+In those cases, these teams may create sub teams.
```

Comment:
How are these sub teams created? Is it by an RFC (or an amendement to an existing RFC), or a vote of its parent (with or without a defining document)? Are teams free to have any number of subteams with any level of nesting?

Reply:
As part of the RFC if one exists (you'll notice that a decent amount of historical teams don't have RFCs), nesting can be infinite, there is no need to have a hard limit of them